### PR TITLE
Add foobar2000

### PR DESCRIPTION
**Pull request by boarnoah**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/518)

### CODE REVIEW

**Comment by boarnoah** [comment]
This package doesn't have a silentwithprogress flag, however ci pipeline insisted on its presence.

File: manifests/foobar2000/foobar2000/1.5.4.Yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Description: foobar2000 music player
+Homepage: https://www.foobar2000.org/
+InstallerType: exe
+Installers:
+  - Arch: x86
+    Url: http://www.foobar2000.org/files/23ded734ebe6fd79f0ce3e50ee03cc84/foobar2000_v1.5.4.exe
+    Sha256: 838ADC84C9D36312F82E333C8876CE9B653436F33912A00F879E52F99A1C8654
+    Switches:
+      Silent: /S
+      SilentWithProgress: /S
```
Comment (superusercode) [inline manifests/foobar2000/foobar2000/1.5.4.Yaml:18]:
```suggestion
InstallerType: nullsoft
Installers:
  - Arch: x86
    Url: http://www.foobar2000.org/files/23ded734ebe6fd79f0ce3e50ee03cc84/foobar2000_v1.5.4.exe
    Sha256: 838ADC84C9D36312F82E333C8876CE9B653436F33912A00F879E52F99A1C8654
```

Comment (superusercode) [reply manifests/foobar2000/foobar2000/1.5.4.Yaml:18]: This is a nullsoft installer, so the nullsoft type can be used. No flags are necessary for the nullsoft type.

**Comment by boarnoah** [comment]
I am closing this as it is not possible to find and official permanent link to the installer, the current links have a "session" of some sort in the URL which expires after about a day (why the hash verification fails).

**Pull request closed without merging**